charts/ghga-common/templates/_image-pull-secrets.tpl

```diff
@@ -0,0 +1,6 @@
+{{- define "ghga-common.image-pull-secrets" -}}
+{{- if .Values.imagePullSecretName -}}
+imagePullSecrets:
+  - name: {{ .Values.imagePullSecretName }}
```

Comment:
Please make this more generic, like this we can only add one secret (which is fine for the moment). However, the requirements may change in the future. Check out [this example](https://stackoverflow.com/questions/62901288/how-to-use-imagepullsecrets-in-helm-3).